operator/internal/manifests/internal/config/build_test.go

```diff
@@ -216,6 +211,15 @@ overrides:
 			Directory:             "/tmp/wal",
 			IngesterMemoryRequest: 5000,
 		},
+		ObjectStorage: storage.Options{
+			S3: &storage.S3StorageConfig{
+			Endpoint:        "http://test.default.svc.cluster.local.:9000",
```

Comment:
This does not seem to be properly indented. It's the same multiple times in this file, but the formatter should fix all of them.